nginx.conf

```diff
@@ -24,9 +24,30 @@ http {
    access_log /dev/stdout;
    root /app;
 
-   location / {
+   location /business {
      alias /app;
      try_files $uri /index.html;
+     gzip on;
+     gzip_vary on;
```

Comment:
Nice to have compression enabled! This should help with performance.
Can this be used in other 2 UIs as well?

Also, do you know how to set up caching for static assets? This could be a new ticket. (See Chrome Lighthouse Performance audit for opportunities.)

Reply:
Done. 

You can create a ticket to address the caching things.

Reply:
Thanks!
I've created a new ticket (https://github.com/bcgov/entity/issues/5037).